importScripts/sampleRecords/subjectList.source.json

```diff
@@ -1,1162 +1,559 @@
   "os_ObjectSubjectID": 94,
   "os_Association": "Wife of artist.",
-  "os_Dates": null,
-  "os_DateMin": null,
-  "os_DateMax": null,
   "o_Accession_Full_ID": "A1986.138h",
   "o_ItemType": "ArtWork",
   "ss_SubjectSubjectID": 2,
   "relatedSubject_SubjectID": 104,
   "relatedSubject_SubjectName": "PRINSEP, Henry Charles (Bonchop]",
   "ss_Association": "Josephine is wife of Henry Charles Princep",
-  "ss_Dates": null,
-  "ss_DateMin": null,
-  "ss_DateMax": null,
   "relatedSubject_SubjectType": "Person"
  },
  {
   "s_SubjectID": 68,
-  "s_Dates": null,
-  "s_HeritageSite": null,
   "s_LastEditBy": "IST",
   "s_LastEditDate": "2000-1",
-  "s_LoadSubjectID": null,
-  "s_SubjectDescription": null,
   "s_SubjectName": "Bussell, Josephine",
-  "s_SubjectResearchBy": null,
   "s_SubjectType": "Person",
   "os_ObjectSubjectID": 95,
   "os_Association": "Wife of artist",
-  "os_Dates": null,
-  "os_DateMin": null,
-  "os_DateMax": null,
   "o_Accession_Full_ID": "A1986.138j",
   "o_ItemType": "ArtWork",
   "ss_SubjectSubjectID": 2,
   "relatedSubject_SubjectID": 104,
   "relatedSubject_SubjectName": "PRINSEP, Henry Charles (Bonchop]",
   "ss_Association": "Josephine is wife of Henry Charles Princep",
-  "ss_Dates": null,
-  "ss_DateMin": null,
-  "ss_DateMax": null,
   "relatedSubject_SubjectType": "Person"
  },
  {
   "s_SubjectID": 68,
-  "s_Dates": null,
-  "s_HeritageSite": null,
   "s_LastEditBy": "IST",
   "s_LastEditDate": "2000-1",
-  "s_LoadSubjectID": null,
-  "s_SubjectDescription": null,
   "s_SubjectName": "Bussell, Josephine",
-  "s_SubjectResearchBy": null,
   "s_SubjectType": "Person",
   "os_ObjectSubjectID": 96,
   "os_Association": "wife of artist",
-  "os_Dates": null,
-  "os_DateMin": null,
-  "os_DateMax": null,
   "o_Accession_Full_ID": "A1986.138k",
   "o_ItemType": "ArtWork",
   "ss_SubjectSubjectID": 2,
   "relatedSubject_SubjectID": 104,
   "relatedSubject_SubjectName": "PRINSEP, Henry Charles (Bonchop]",
   "ss_Association": "Josephine is wife of Henry Charles Princep",
-  "ss_Dates": null,
-  "ss_DateMin": null,
-  "ss_DateMax": null,
   "relatedSubject_SubjectType": "Person"
  },
  {
   "s_SubjectID": 68,
-  "s_Dates": null,
-  "s_HeritageSite": null,
   "s_LastEditBy": "IST",
   "s_LastEditDate": "2000-1",
-  "s_LoadSubjectID": null,
-  "s_SubjectDescription": null,
   "s_SubjectName": "Bussell, Josephine",
-  "s_SubjectResearchBy": null,
   "s_SubjectType": "Person",
   "os_ObjectSubjectID": 97,
   "os_Association": "wife of artist",
-  "os_Dates": null,
-  "os_DateMin": null,
-  "os_DateMax": null,
   "o_Accession_Full_ID": "A1986.138l",
   "o_ItemType": "ArtWork",
   "ss_SubjectSubjectID": 2,
   "relatedSubject_SubjectID": 104,
   "relatedSubject_SubjectName": "PRINSEP, Henry Charles (Bonchop]",
   "ss_Association": "Josephine is wife of Henry Charles Princep",
-  "ss_Dates": null,
-  "ss_DateMin": null,
-  "ss_DateMax": null,
   "relatedSubject_SubjectType": "Person"
  },
  {
+  "s_SubjectID": 97,
```

Comment:
This subject has no corresponding destination subject. See row 161 of https://docs.google.com/spreadsheets/d/1zKHsF977x6SlQN7JtvnN-F4mGlNNcMClcwBAT_967zU/edit#gid=1435981909